lib/utils/allFilesIgnoredError.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+class AllFilesIgnoredError extends Error {
+	/**
+	 * @param {string[]} fileList
+	 */
+	constructor(fileList) {
+		super();
+
+		const pattern = fileList.filter((i) => !i.startsWith('!')).join(', ');
+
+		this.message = `File "${pattern}" ignored because of a matching ignore pattern. Move ignore config to .stylelintrc or use "--allow-empty-input" to override.`;
```

Comment:
[suggest] It may be better to wrap each file with double quotes. What do you think?

```suggestion
		const pattern = fileList.filter((i) => !i.startsWith('!')).map((i) => `"${i}"`).join(', ');

		this.message = `File ${pattern} ignored because of a matching ignore pattern. Move ignore config to .stylelintrc or use "--allow-empty-input" to override.`;
```

Reply:
@ybiquitous Thanks for confirming the pull request is heading in the right direction.

Am I right in thinking that this error will only show if all the files matching the input are ignored?

And, if the input glob touches a lot of files, the `fileList` could be long?

Shall we make the error message simple, e.g. "All input files were ignored because of the ignore pattern. Either change your input, ignore pattern or use "--allow-empty-input" to allow no inputs." Rather than have the possibility of outputting a long list of files.

Reply:
@jeddy3 That's a good idea! 👍🏼